src/sidebar/components/markdown-editor.js

```diff
@@ -259,7 +259,7 @@ export default function MarkdownEditor({
   };
 
   const handleKeyDown = event => {
-    if (!event.ctrlKey) {
+    if (!event.ctrlKey && !event.metaKey) {
```

Comment:
It would be nice to adjust the tooltips to show the "normal" shortcut for the action depending on the platform. I don't know the best way to do this however. This shouldn't block merging this change however.

Reply:
Yep! This was something that occurred to me. GitHub, e.g., shows me the modifier key relevant to my OS (`cmd`) in their editor which I'm typing in. Depending on how valuable you think this is, I could do some Googling around to see if there is a quick and dirty way to sniff for this. I mean, we could do something hamfisted like look at an OS string and parse it for Mac, but I don't know if that's a game we want to play?

Reply:
Hi Lyza,

It seems that GitHub sniffs the user agent. Following [these steps](https://developers.google.com/web/tools/chrome-devtools/device-mode/override-user-agent) to change the OS to Windows results in me seeing "Ctrl+<key>" shortcut hints. Defaulting to "Ctrl+<key>" in the tooltip but changing to "Cmd+<key>" if the user agent contains "Mac OS X" (this covers [both macOS and iOS](https://www.whatismybrowser.com/guides/the-latest-user-agent/safari)) would be a nice polish detail. If we continue to allow both modifiers then we'll degrade gracefully if the UA sniffing result is wrong for some reason.

Reply:
Bah, should have pushed updates to this branch before leaving for the weekend; I have indeed made changes similar to to this to my local branch.